doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -37,6 +37,105 @@ It is recommended to follow the “url” link values,
 applicable to retrieve resources. Instead of constructing your own URLs,
 to keep your client decoupled from implementation details.
 
+## JSON entity representation
+
+Many operations use the same JSON entity representation format which is described in the present section. Each
+entity is represented by a JSON object which fields are the following ones:
+
+* `id` and `type` are the entity ID and entity type, respectively. In the case the entity has an attribute named
+  `id` and/or `type` then they are ignored. For entities without type (i.e. type equals to "" internally), the
+  `type` field is not included
+* All other first level fields correspond to the names of the attributes of the entity.
+  * If the attribute type allows "compact form" and the attribute doesn't have any metadata element, the value of the
+    attribute field is the attribute value itself.
+  * If the attribute type doesn't allow "compact form" and/or the attribute have at least a metadata element, the value
```

Comment:
I think there is no need to coined the term "compact form" as this is the regular way JSON format works